contracts/StorageContract.sol

```diff
@@ -81,6 +81,9 @@ abstract contract StorageContract is DecentralizedKV {
     /// @notice
     uint256 public prepaidLastMineTime;
 
+    /// @notice Fund tracker for prepaid
+    uint256 public totalPrepaidAmount;
```

Comment:
Another variable name we could consider is “accPrepaidAmount” for the accumulative prepaid amount.